samples/cellular/nrf_cloud_multi_service/overlay_nrf7002ek_wifi_coap_no_lte.conf

```diff
@@ -76,6 +76,17 @@ CONFIG_DATE_TIME_MODEM=n
 CONFIG_DATE_TIME_NTP=y
 CONFIG_DATE_TIME_LOG_LEVEL_DBG=y
 
+## Enable date_time fast-retries for Wi-Fi
+## This prevents occasional DNS lookup flukes from
+## rendering the device unable to connect until the next
+## DATE_TIME_INTERVAL_SECONDS elapses (default 4 hours)
+CONFIG_DATE_TIME_NTP_FAST_RETRY=y
```

Comment:
Isn't this still the wrong name? `CONFIG_DATE_TIME_NTP_FAST_RETRY`

Reply:
Yep, sorry, I missed updating the config files, my bad